### PR TITLE
Implement numberless chapter resolving and fix error messages.

### PR DESCRIPTION
**Pull request by Ivanmatthew**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # MangaStream Extensions
-[Click me](https://thenetsky.github.io/extensions-generic-0.8/) to go to the index page!
+[Click me](https://ivanmatthew.github.io/netsky-fork-generic/) to go to the index page!
```
Comment (TheNetsky) [inline README.md:2]: ?

File: src/MangaStream.ts [hunk 2]
```diff
@@ -36,7 +36,7 @@ import {
 } from './MangaStreamInterfaces'
 
 // Set the version for the base, changing this version will change the versions of all sources
-const BASE_VERSION = '3.0.3'
+const BASE_VERSION = '3.1.3'
```
Comment (TheNetsky) [inline src/MangaStream.ts:39]: You mean either 3.1.0 or 3.0.4?

**Comment by TheNetsky** [comment]
This definitely needs some testing before hand, not sure for which source or in which case this is used or need to be used?

File: README.md (hunk 1 above)
Comment (Ivanmatthew) [reply README.md:2]: Forgot it wasn't the gh-pages branch

File: src/MangaStream.ts (hunk 2 above)
Comment (Ivanmatthew) [reply src/MangaStream.ts:39]: Reflected in new PR

**Comment by Ivanmatthew** [comment]
> This definitely needs some testing before hand, not sure for which source or in which case this is used or need to be used?

Use-case is for MangaGalaxy, see support discord channel.
Example is chapter 0 of https://mangagalaxy.me/series/worn-and-torn-newbie-13/

Has already been tested to be working.

New PR where this is scoped for MangaGalaxy only: https://github.com/TheNetsky/extensions-generic-0.8/pull/96

Initially thought it was behavior that could reflect on all generics, so implemented it generic wide instead of source scoped.

**Pull request closed without merging**